config.json.example

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
delete unused config file since we're using env vars

Reply:
I thought we could keep both options, but I'll switch everything up to envvars.